### PR TITLE
add Run Jin, Shannon Robbins, update Krutika's email

### PR DESCRIPTION
**Pull request by jharenza**
### Purpose

- add Run Jin, Shannon Robbins as authors
- update Krutika's email

### Issues

#140 
#168 

#### Which areas should receive a particularly close look?
NA

#### Is there anything that you want to discuss further?
NA

#### Is the pull request ready for review?
yes

### Pull review checklist

Unless otherwise noted above, this PR will be considered ready for review when all four items have been checked.

- [x] All changes to text follow "one sentence per line" [[Manubot instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#manubot-markdown)]
- [x] All citations follow the [Manubot citation instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#citations)
- [x] All changes or additions to tables follow the [Manubot table instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#tables)
- [x] All figures follow the [Manubot figure instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#figures)
- [x] There are no new spelling errors identified by the [Manubot spellchecker](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#spellchecking)

### Spellcheck Step

The dictionary used for spellchecking can be updated.
Edit the file in `build/assets/custom-dictionary.txt` by adding new entries to the end.
You do not need to change anything else.
However, if you want to update the first line to have an accurate count of words and you want to remove non-unique ones, run the following command from within `build/assets` on your favorite OS X or Linux machine:
```
(( len = $(awk '!a[$0]++' < custom-dictionary.txt | wc -l ) - 1 )); tmpfile="$(mktemp)"; echo "personal_ws-1.1 en $len utf-8" > $tmpfile; tail -n +2 custom-dictionary.txt | awk '!a[$0]++' >> $tmpfile; mv $tmpfile custom-dictionary.txt
```

### CODE REVIEW

**Comment by jharenza** [comment]
hmm @jaclyn-taroni - put in this quick PR and manubot is failing. I tried a rerun, but failed again. Not sure why..

**Comment by jaclyn-taroni** [comment]
Fixed in #181. So if you want to approve that, I can merge it and then pull `master` into this branch.

**Comment by sjspielman** [comment]
One possible issue here - since we're updating Krutika's email, I am seeing her given twitter handle `29aprilkrutika` doesn't exist - https://twitter.com/29aprilkrutika 
@kgaonkar6 , do you have another twitter, or should we remove your handle here altogether?

**Comment by jharenza** [comment]
> One possible issue here - since we're updating Krutika's email, I am seeing her given twitter handle `29aprilkrutika` doesn't exist - https://twitter.com/29aprilkrutika @kgaonkar6 , do you have another twitter, or should we remove your handle here altogether?

oh, sneaky @kgaonkar6  - she updated it to `@aggokittu` :P - i will update that now!

File: content/metadata.yaml [hunk 1]
```diff
@@ -65,8 +65,8 @@ authors:
     name: Krutika S. Gaonkar
     initials: KSG
     orcid: 0000-0003-0838-2405
-    twitter: 29aprilkrutika
-    email: gaonkark@chop.edu
+    twitter: @aggokittu
```
Comment (sjspielman) [inline content/metadata.yaml:68]: No `@` and should be good -

```suggestion
    twitter: aggokittu
```

Comment (jharenza) [reply content/metadata.yaml:68]: oh shoot

**Review by sjspielman — APPROVED** [review]

**Comment by sjspielman** [comment]
Looks good!

**Pull request merged**